client/rest/resources/rest.json

```diff
@@ -14,7 +14,9 @@
   },
   "uncirculatingAccountPublicKeys": [
     "A4739036FD7EFED2750A51EE9D1D3113BA3F9849E0889213CED7F221B2AA1A20",
-    "2BF1E1F3072E3BE0CD851E4741E101E33DB19C163895F69AA890E7CF177C878C"
+    "2BF1E1F3072E3BE0CD851E4741E101E33DB19C163895F69AA890E7CF177C878C",
+    "E85D10BF47FFBCE2230F70CB43ED2DDE04FCF342524B383972F86EA1FF773C79",
```

Comment:
does node.js allow comments within json files? (it'd be quite helpful here...)

Reply:
JSON does not allow comments in `//` form. It's possible in the JSON5 subset, which is not used here. I could add a property like "_comment" before this uncirculatingAccountPublicKeys, but not sure if it will be readable.